conf/app.routes

```diff
@@ -42,6 +42,6 @@ GET         /start                                         controllers.StartCont
 
 # Declaration Choice Page
 
-GET         /choice                                        controllers.ChoiceController.displayPage()
+GET         /choice                                        controllers.ChoiceController.displayPage(previousChoice: Option[String] ?= None)
```

Comment:
Can we add `QueryStringBindable` for `Choice` ? 

I am thinking about usage of Null Object Pattern here with object `NoChoice` as explicit null but maybe it is too much complication ?

Reply:
I agree! (That it's too much complication)

Reply:
Depending of which code - little more complicated at at `Choice` definition but all clients are not have to deal with Option everywhere

Reply:
I've implemented Lukasz'z suggestion, if you want to take a 2nd look